docs/content/configuration/basics.md

```diff
@@ -0,0 +1,650 @@
+```
+
+Or for each connection:
+
+```yml
+# db_tools.yaml
+connections:
+    connection_one:
+        backup_excluded_tables: ['table1', 'table2']
+    connection_two:
+        backup_excluded_tables: ['table3', 'table4']
+```
+
+</div>
+
+When working with multiple connections, any connection which does not specify the option
+will inherit from the default.
+:::
+
+## Backup configuration
+
+Some options are available to customize how the `db-tools:backup` command works.
+
+### Storage
+
+#### Root directory
+
+The `db_tools.storage.root_dir` parameter let you choose where to put the generated dumps.
+
+<div class="symfony">
+
+Default value is `'%kernel.project_dir%/var/db_tools'`.
+
+</div>
+
+<div class="symfony">
+
+Default value is `./var/db_tools'`.
+
+</div>
+
+#### File and directory naming strategy
+
+Default behavior will store your backup using this strategy:
+`%db_tools.storage.root_dir%/<YEAR>/<MONTH>/<CONNECTION-NAME>-<YEAR><MONTH><DAY><HOUR><MINUTES><SECOND>.<EXT>`
+where `<EXT>` is the file extension depending upon the database backend (mostly `.sql` or `.dump`).
+
+<div class="symfony">
+
+Custom strategy can be implemented by extending the
+`MakinaCorpus\DbToolsBundle\Storage\AbstractFilenameStrategy` abstract class:
+
+```php
+namespace App\DbTools\Storage;
+
+use MakinaCorpus\DbToolsBundle\Storage\AbstractFilenameStrategy;
+
+class FooFilenameStrategy extends AbstractFilenameStrategy
+{
+    #[\Override]
+    public function generateFilename(
+        string $connectionName = 'default',
+        string $extension = 'sql',
+        bool $anonymized = false
+    ): string {
+        return '/some_folder/' . $connectionName . '.' . $extension;
+    }
+}
+```
+
+Then registered this way, on a per-connection basis:
+
+```yaml
+# config/packages/db_tools.yaml
+db_tools:
+    storage:
+        filename_strategy:
+            connection_name: App\DbTools\Storage\FooFilenameStrategy
+```
+
+Value can be a container service identifier, or directly a class name in case this
+has no constructor arguments.
+
+If you need to store your dumps outside of the `%db_tools.storage.root_dir%` directory,
+then implement the `MakinaCorpus\DbToolsBundle\Storage\FilenameStrategyInterface` directly
+and add the following method:
+
+```php
+namespace App\DbTools\Storage;
+
+use MakinaCorpus\DbToolsBundle\Storage\FilenameStrategyInterface;
+
+class FooFilenameStrategy implements FilenameStrategyInterface
+{
+    #[\Override]
+    public function generateFilename(/* ... */): string {}
+
+    #[\Override]
+    public function getRootDir(
+        string $defaultRootDir,
+        string $connectionName = 'default',
+    ): string {
+        return '/some/path/' . $connectionName . '/foo';
+    }
+}
+```
+
+This will allow the restore command to find your backups.
+
+</div>
+
+<div class="standalone">
+
+:::warning
+There is as of now no way to implement a custom filename strategy when using *DbToolsBundle* as a standalone
+CLI tool.
+:::
+
+</div>
+
```

Comment:
May be we can add an :::info telling people that other FilenameStrategy could be added in the future and that they can ask for them if they have idea or something

Reply:
```
:::info
More filename strategies may be implemented in core in the future. If you have any
suggestions, please [open an discussion](https://github.com/makinacorpus/DbToolsBundle/issues) about it.
:::
```

Is good ?